WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/wooshippinglabels/models/ShippingLabelModel.kt

```diff
@@ -25,7 +29,18 @@ data class ShippingLabelModel(
     val rate: BigDecimal,
     val currency: String,
     val expiryDate: Long,
-)
+) : Parcelable {
+    val isRefundAvailable: Boolean
+        get() {
+            val createdTime = createdDate?.time ?: return true
+            return status != ShippingLabelStatus.ANONYMIZED &&
+                Date().before(Date(createdTime + TimeUnit.DAYS.toMillis(REFUND_EXPIRY_DAYS))) == true
```

Comment:
This logic was copied from [ShippingLabelRefundViewModel](https://github.com/woocommerce/woocommerce-android/blob/0d26c454ff1b17d55bae755ad1091a20d4094e38/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/ShippingLabelRefundViewModel.kt#L82)

Reply:
np, I know I'm the one who wrote the logic in the old flow, but now that I'm thinking about it, I think using `Date` for this is kinda an overkill, WDYT about something simpler like:

```suggestion
            val createdTime = createdDate?.time ?: return true
            return status != ShippingLabelStatus.ANONYMIZED &&
                System.currentTimeMillis() > createdTime + TimeUnit.DAYS.toMillis(REFUND_EXPIRY_DAYS)
```

The above avoids two instantiations, and to me the readability is not impacted at all.

Reply:
Updated: 09141d9072beea704c422452f4ced2d7b860ada7
Also added test for `isRefundAvailable`